apps/condo/domains/news/schema/OrganizationNewsItem.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Generated by `createschema news.OrganizationNewsItem 'organization:Relationship:Organization:CASCADE; title:Text; body:Text; type:Select:common,emergency'`
+ */
+
+const { Text, Relationship, Select } = require('@keystonejs/fields')
+
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/news/access/OrganizationNewsItem')
+const { NEWS_TYPES } = require('@condo/domains/news/constants')
+
+const OrganizationNewsItem = new GQLListSchema('OrganizationNewsItem', {
+    schemaDoc: 'The news item created by the organization to show on resident\'s mobile devices',
+    fields: {
+
```

Comment:
As I understant, targets, publication date, etc. will be added later ?

Reply:
Yes, right